app/src/main/java/app/accrescent/client/ui/SettingsScreen.kt

```diff
@@ -194,9 +194,13 @@ fun Setting(
             modifier = Modifier
                 .padding(end = 8.dp)
                 .weight(1f),
+            horizontalAlignment = Alignment.Start,
+            verticalArrangement = Arrangement.Center,
```

Comment:
What is the effect of deviating from the defaults like this? I don't see a difference on either Pixel 3a or Pixel Tablet form factors.

Reply:
Oh oops, this is redundant since the parent composable does the alignment that I was worried about here. I have reverted this change.